bookkeeper-server/src/test/java/org/apache/bookkeeper/client/GenericEnsemblePlacementPolicyTest.java

```diff
@@ -132,9 +132,9 @@ public void testReplaceBookie() throws Exception {
                     lh.addEntry(value);
                 }
             }
-            assertEquals(2, customMetadataOnNewEnsembleStack.size());
```

Comment:
@sijie in usecases covered by me and my company I am already handling this problem by falling  back to DefaultEnsamblePlacementPolicy in case for non-compatibility.

So I am tempted to close the issue as "won't fix".
I wonder if other companies will have such problems, we should make a clear list of changes and maybe a guide to the migration to 4.5.
As you previously said this would be a 5.0 release, but let's stick to 4.5, we want to close as soon as possible

are you ok with closing the issue as won't fix ?